text/0032-nft-standard.md

```diff
@@ -69,14 +71,49 @@ The metadata should have the following JSON structure:
         "type": "string",
         "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
     },
+    "collection": {
+        "type": "Object<CollectionObject>",
+        "description": "Additional details are recommended for NFT collections with multiple unique assets under one single family and symbol."
+    },
     "attributes": {
         "type": "Array<AttributeObject>",
         "description": "These are the extra attributes for the digital asset. It's an array of AttributeObject to make it as flexible as possible."
     },
+    "royalty": {
+        "type": "Object<RoyaltyObject>",
+        "description": "Object with fields that would be utilised to pay royalty to original creators in secondary sales conducted by marketplaces that adheres Hathor Metadata standards, after minting (primary sale)."
```

Comment:
to pay royalties

adheres to Hathor Metadata standards

Reply:
Done